CONTRIBUTING.md

```diff
@@ -38,10 +38,11 @@ $ cabal run cabal -- build --help
 ```
 
 > [!NOTE]
-> If you're using Nix, you might find it convenient to work within a shell that has all the `Cabal` development dependencies:
-> ```
-> $ nix-shell -p cabal-install ghc ghcid haskellPackages.fourmolu_0_12_0_0 pkgconfig zlib.dev
+> If you're using Nix, you might find it convenient to work within a shell that has the following `Cabal` development dependencies:
+> ```bash
+> $ nix-shell -p cabal-install ghc ghcid pkg-config zlib.dev # incomplete
 > ```
+> One dependency that we left out in the above command is `haskellPackages.formolu_0_12_0_0` which would need to be installed manually.
```

Comment:
We use fourmolu 0.14.0.0

Reply:
I don't think so: https://github.com/haskell/cabal/blob/72e54398ab40d0b97d9f342d35c4d6d3128d1736/.github/workflows/format.yml#L15

Reply:
Well-spotted, I was under a false impression.

Reply:
@taimoorzaeem nevermind then, apologies for the confusion!